src/sparseml/transformers/finetune/data/custom.py

```diff
@@ -55,9 +59,22 @@ def get_raw_dataset(self, *_ignore, **__ignore) -> Union[DatasetDict, Dataset]:
             raw_dataset = super().get_raw_dataset()
 
         if self.preprocessing_func is not None:
+
+            if isinstance(self.preprocessing_func, Callable):
+                func = self.preprocessing_func
+            else:
+                if ":" in self.preprocessing_func:
+                    # load func_name from "/path/to/file.py:func_name"
+                    func = import_from_path(self.preprocessing_func)
```

Comment:
https://github.com/neuralmagic/sparsezoo/blob/eabb28ca0b8d6a14089f5967c14457237daf04d1/src/sparsezoo/utils/registry.py#L258 Don't we already have support of this in the above line? If that would would we could combine the two if conditions maybe?